### PR TITLE
Set default mode for TLS files

### PR DESCRIPTION
**Pull request by stanlyDoge**
### Type of change
- Enhancement / new feature

### Description

Set default mode for TLS files.

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by stanlyDoge** [comment]
Any idea why `testSendMessagesPlainAnonymous` keeps failing (timeout) with this change?

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java [hunk 1]
```diff
@@ -641,6 +641,7 @@ protected ConfigMap createConfigMap(String name, Map<String, String> data) {
     protected Volume createSecretVolume(String name, String secretName) {
 
         SecretVolumeSource secretVolumeSource = new SecretVolumeSourceBuilder()
+                //.withDefaultMode(0440)
```
Comment (scholzj) [inline cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java]: Why is the default mode commented out?

Comment (stanlyDoge) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java]: Sorry I was just trying whether this change has an influence on failing test (I was unable to run it locally).

**Comment by stanlyDoge** [comment]
@ppatierno @tombentley Can you please do a new review? I had to change a few files.

**Pull request merged**